resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -60,1404 +60,1412 @@
+          label: Biomarkers/VEGF receptor score
+          type: basic:decimal
+          description: |
+            Human vascular endothelial growth factor receptor e.g. VEGFR-1 (Flt-1) and VEGFR-2 (KDR/Flk-1).
+          required: false
+        - name: biomarkers__itgb3
+          label: Biomarkers/ITGB3
+          type: basic:decimal
+          required: false
+        - name: biomarkers__MLH1_loss_score
+          label: Biomarkers/MLH1 loss score
+          type: basic:decimal
+          required: false
+        - name: biomarkers__hrd_loh
+          label: Biomarkers/HRD-LOH
+          type: basic:decimal
+          required: false
+        - name: biomarkers__ctdna
+          label: Biomarkers/ctDNA
+          type: basic:decimal
+          required: false
+        - name: biomarkers__cps
+          label: Biomarkers/CPS
+          type: basic:decimal
+          description: |
+            Combined positive score.
+          required: false
+        - name: biomarkers__tmb_score
+          label: Biomarkers/TMB
+          type: basic:integer
+          description: |
+            Tumor mutation burden - the total number of genomic alterations
+            found in the cells of a cancer.
+            See http://purl.obolibrary.org/obo/NCIT_C150128.
+          required: false
+        - name: biomarkers__tmb_classification
+          label: Biomarkers/TMB assessment
+          type: basic:string
+          choices:
+            - label: High
+              value: high
+            - label: Medium
+              value: medium
+            - label: Low
+              value: low
+          description: |
+            Tumor mutation burden assessment - a quantitative or qualitative
+            assessment of the number of genomic alterations found in genetic
+            material from tumor cells, when compared to a reference sequence.
+            See http://purl.obolibrary.org/obo/NCIT_C181335.
+          required: false
+        - name: biomarkers__pdl1_cps
+          label: Biomarkers/PDL1 CPS
+          type: basic:integer
+          description: |
+            PDL1 combined positive score (CPS).
+          required: false
+        - name: biomarkers__pdl1_tps
+          label: Biomarkers/PDL1 TPS
+          type: basic:integer
+          description: |
+            PDL1 tumor proportion score (TPS).
+          required: false
+        - name: biomarkers__pdl1_tps_cat
+          label: Biomarkers/PDL1 TPS category
+          type: basic:string
+          choices:
+            - label: <1%
+              value: less_than_1
+            - label: 1-49%
+              value: from_1_to_49
+            - label: 50%<
+              value: more_than_50
+            - label: N/A
+              value: na
+          description: |
+            PDL1 tumor proportion score category.
+          required: false
+        - name: biomarkers__chip
+          label: Biomarkers/CHIP
+          type: basic:integer
```

Comment:
Based on the data this looks like a binary field with values 0/1. IMO it would make sense to have this as a string field with two choices.
```
choices:
    - label: "Yes"
      value: "yes"
    - label: "No"
      value: "no"
```

Reply:
@OtonicarJan @AnzeLovse agree